src/clj_kondo/impl/analyzer.clj

```diff
@@ -982,6 +983,10 @@
                           var-name
                           expr
                           (assoc (meta expr)
+                                 :name-row (:row (meta var-name))
```

Comment:
It looks good, but I think you could only call `meta` once and store that in a local.

Reply:
Done!